### PR TITLE
Fixing coms

### PR DESCRIPTION
**Pull request by Shoxer90**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 import './App.css';
 
-const App = () => {
-  return (
-    <div>
-      <Component />
-    </div>
+const App = () => (
+    <DndProvider backend={HTML5Backend}>
+      <Component/>
```
Comment (Suren-Sargsyan14) [inline src/App.js]: If your component is closing itself and it contains just one line the closing "/" need to have one space before it, like

<Component />

File: src/Components/Buttons/index.js [hunk 2]
```diff
@@ -2,14 +2,11 @@ import React from "react";
 
 import styles from "./index.module.scss";
 
-const Button = ({name,func,h,w}) => {
-
-    const handleClick = () => {
-        return func();
-    };
+const Button = ({name, func}) => {
+    const handleClick = () => func();
```
Comment (Suren-Sargsyan14) [inline src/Components/Buttons/index.js]: This line is unnecessary, please remove it.

File: src/Components/Buttons/index.js [hunk 3]
```diff
@@ -2,14 +2,11 @@ import React from "react";
 
 import styles from "./index.module.scss";
 
-const Button = ({name,func,h,w}) => {
-
-    const handleClick = () => {
-        return func();
-    };
+const Button = ({name, func}) => {
```
Comment (Suren-Sargsyan14) [inline src/Components/Buttons/index.js]: Please add spaces between curly brackets, if the object or destructor contains only one line.

File: src/Components/Buttons/index.js [hunk 4]
```diff
@@ -2,14 +2,11 @@ import React from "react";
...
+    const handleClick = () => func();
     
     return (
-        <button className={styles.button} style={{height:h,width:w}}  onClick={()=>handleClick()}>
+        <button className={styles.button} onClick={()=>handleClick()}>
```
Comment (Suren-Sargsyan14) [inline src/Components/Buttons/index.js]: this syntax is unnecessary and wrong with logic, try to do this very little time you can just write onClick={handleClick}

File: src/Components/Buttons/index.module.scss [hunk 5]
```diff
@@ -2,8 +2,8 @@
...
     background-color: green;
     border: solid rgb(4, 68, 4) 1px;
-    border-radius: 13px;
-    padding: 5px 20px;
-    font-size: 120%;
+    border-radius: 5px;
+    padding: 5px 5px;
+    width: fit-content;
     align-self: center;
 }
```
Comment (Suren-Sargsyan14) [inline src/Components/Buttons/index.module.scss]: please add a new line in each end of file

File: src/Components/Contacts/Avatar.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+import React, { memo } from "react";
+
+const Avatar = ({src,className,alt}) => (<img className={className} src={src} alt={alt}/>);
```
Comment (Suren-Sargsyan14) [inline src/Components/Contacts/Avatar.js]: Please add spaces between curly brackets, if the object or destructor contains only one line.

also you need to write a component like this

```
const Avatar = ({ src, className, alt }) => (
  <img
     className={className}
     src={src}
     alt={alt}
  />
);

```

File: src/Components/Contacts/Contact.js [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+    email,
+    firstName,
+    removeContact}) => {
+        
+    const [showMore, setShowMore] = useState(false);
+    const [showDownloader, setShowDownloader] = useState(false);
+    const [userPhoto, setUserPhoto] = useState([]);
+
+
+    const setPhotosToFB = async(fromFileInput) => {
```
Comment (Suren-Sargsyan14) [inline src/Components/Contacts/Contact.js]: please set a space after async

File: src/Components/Contacts/Contact.js [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+        const arrOfImg=[] 
+        fromFileInput.map((photo)=>{
```
Comment (Suren-Sargsyan14) [inline src/Components/Contacts/Contact.js]: please set spaces before and after "=>" sign
also before curly bracket

File: src/Components/Contacts/Contact.js [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+    firstName,
+    removeContact}) => {
+        
+    const [showMore, setShowMore] = useState(false);
+    const [showDownloader, setShowDownloader] = useState(false);
+    const [userPhoto, setUserPhoto] = useState([]);
+
+
+    const setPhotosToFB = async(fromFileInput) => {
+        const arrOfImg=[]
```
Comment (Suren-Sargsyan14) [inline src/Components/Contacts/Contact.js]: please set spaces before and after = sign

File: src/Components/Contacts/Contact.js [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+
+    const setPhotosToFB = async(fromFileInput) => {
+        const arrOfImg=[] 
+        fromFileInput.map((photo)=>{
+            const  photoId = uuidv4();
+            const imageRef = ref(fireStorage, `${id}/ ${photoId}`);
+            uploadBytes(imageRef, photo).then(() => {
+                const imageListRef = ref(fireStorage, `${id}/`);
+                listAll(imageListRef).then((response) => {
+                    let filterData = response.items.filter((item) =>item.name === photoId);
```
Comment (Suren-Sargsyan14) [inline src/Components/Contacts/Contact.js]: please set space after "=>" sign

File: src/Components/Contacts/Contact.js [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+                });
+            });
+        });
+        setUserPhoto(arrOfImg);
+        setPhotosToUserData(userPhoto, id);
+    };
+
+    const handleOpenInfo = () => {
+        setShowMore(!showMore);
+        setShowDownloader(showDownloader => showDownloader = false);
```
Comment (Suren-Sargsyan14) [inline src/Components/Contacts/Contact.js]: you can just write `setShowDownloader(false);`

File: src/Components/Contacts/Contact.js [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+    };
+
+    const handleOpenDownloader = () => setShowDownloader(!showDownloader);
+
+    const deleteContact = () => removeContact(id,avatar);
```
Comment (Suren-Sargsyan14) [inline src/Components/Contacts/Contact.js]: please set space after semicolon

File: src/Components/Contacts/ImageCarret.js [hunk 13]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import Avatar from "./Avatar";
+
+const ImageCarret = ({styles, userPhoto}) => (
```
Comment (Suren-Sargsyan14) [inline src/Components/Contacts/ImageCarret.js]: Please add spaces between curly brackets, if the object or destructor contains only one line.

File: src/Components/Contacts/ImageCarret.js [hunk 14]
```diff
@@ -0,0 +1,13 @@
...
+    <div>
+        {userPhoto.map((photo) => (
+            <Avatar src={photo} className={styles.userImg} alt={new Date()} key ={photo} />
```
Comment (Suren-Sargsyan14) [inline src/Components/Contacts/ImageCarret.js]: if your component has more then 2 props please change it into multiline, and don't set space before and after "=" sign in props

File: src/Components/Contacts/ImageCarret.js [hunk 15]
```diff
@@ -0,0 +1,13 @@
...
+        ))}
+    </div>
+);
+
+export default ImageCarret;
```
Comment (Suren-Sargsyan14) [inline src/Components/Contacts/ImageCarret.js]: Please memoize this component

File: src/Components/Contacts/index.js [hunk 16]
```diff
@@ -0,0 +1,13 @@
+import React, { memo } from "react";
+
+import Contact from "./Contact";
+
+const Contacts = ({ contacts,removeContact }) => (
```
Comment (Suren-Sargsyan14) [inline src/Components/Contacts/index.js]: please set space after semicolon

File: src/Components/Contacts/index.module.scss [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+    background-color: rgb(79, 250, 79);
+    height: 200px;
+    width: auto;
+    overflow-x: scroll;
+}
+.userImg{
+    border: solid pink;
+    height: 50px;
+    width: 50px;
+    >img{
```
Comment (Suren-Sargsyan14) [inline src/Components/Contacts/index.module.scss]: I think you need to write &>img

File: src/Components/FileInput.js/DrowImages.js [hunk 18]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const DrowImages = ({image, style}) => (
```
Comment (Suren-Sargsyan14) [inline src/Components/FileInput.js/DrowImages.js]: Please add spaces between curly brackets, if the object or destructor contains only one line.

File: src/Components/FileInput.js/DrowImages.js [hunk 19]
```diff
@@ -0,0 +1,7 @@
...
+    <img className={style.imageAploader} src={image} alt={new Date()}/>
```
Comment (Suren-Sargsyan14) [inline src/Components/FileInput.js/DrowImages.js]: if your component contains only one line PLEASE set SPACE before `/>`

File: src/Components/FileInput.js/index.js [hunk 20]
```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+
+import { v4 as uuidv4 } from 'uuid';
+
+import DrowImages from "./DrowImages";
+
+import style from "./index.module.scss";
+
+
+const FileInput = ({avatar, setPhotosToFB}) => {
```
Comment (Suren-Sargsyan14) [inline src/Components/FileInput.js/index.js]: Please add spaces between curly brackets, if the object or destructor contains only one line.

File: src/Components/FileInput.js/index.js [hunk 21]
```diff
@@ -0,0 +1,57 @@
...
+        e.preventDefault();
+        setPhotosToFB(Array.from(e.target[0].files), avatar);
+    };
+
+    return (
+        <div className={style.fileLoaderComponent}>
+            {currentImages && 
+                currentImages.map((item) => (
+                    <span key={uuidv4()}>
+                        <DrowImages  image={item} style={style} />
```
Comment (Suren-Sargsyan14) [inline src/Components/FileInput.js/index.js]: Please remove a unnecessary space

File: src/Components/FileInput.js/index.js [hunk 22]
```diff
@@ -0,0 +1,57 @@
...
+                    </span>
+                ))
+            }
+            <span>Drag an image,or load from device</span>
+            <form onSubmit={(e)=>handleSetPhotosToFB(e)}>
```
Comment (Suren-Sargsyan14) [inline src/Components/FileInput.js/index.js]: please set spaces before and after "=>" sign. and you can just write ` <form onSubmit={handleSetPhotosToFB}>`

File: src/Components/FileInput.js/index.js [hunk 23]
```diff
@@ -0,0 +1,57 @@
...
+                <input type="file"
+                    multiple
+                    className={style.hiddenInputFiles}
+                    onDragOver={(e)=>{
+                        e.preventDefault()
+                        e.stopPropagation()
+                    }}
+                    onDrop={()=>onFileLoad}
```
Comment (Suren-Sargsyan14) [inline src/Components/FileInput.js/index.js]: I think this function will not call ever.

You can just write `onDrop={onFileLoad}`

File: src/Components/FileInput.js/index.js [hunk 24]
```diff
@@ -0,0 +1,57 @@
...
+                    onChange={(e)=>onFileLoad(e)}
```
Comment (Suren-Sargsyan14) [inline src/Components/FileInput.js/index.js]: here you can just write `onChange={onFileLoad}`

File: src/Components/FileInput.js/index.js [hunk 25]
```diff
@@ -0,0 +1,57 @@
...
+                />
+                <div>
+                    <button type="submit">Add photos</button>
+                </div>
+            </form>
+        </div>
+    );
+};
+
+export default FileInput;
```
Comment (Suren-Sargsyan14) [inline src/Components/FileInput.js/index.js]: Please memoize this component

File: src/Components/FileInput.js/index.module.scss [hunk 26]
```diff
@@ -0,0 +1,39 @@
...
+    height: 300px;
+    width: 90%;
+    background-color: #eee;
+    border-radius: 10px;
+    border: solid #333;
+    text-align: center;
+    color: #333;
+    align-items: center;
+    margin: 20px;
+    >button{
```
Comment (Suren-Sargsyan14) [inline src/Components/FileInput.js/index.module.scss]: I think you need to write `&>button {`

File: src/Components/FileInput.js/index.module.scss [hunk 27]
```diff
@@ -0,0 +1,39 @@
...
+        position: sticky;
+        margin-top: 22%;
+    }
+}
+.hiddenInputFiles{
```
Comment (Suren-Sargsyan14) [inline src/Components/FileInput.js/index.module.scss]: please add space before {

File: src/Components/FileInput.js/index.module.scss [hunk 28]
```diff
@@ -0,0 +1,39 @@
...
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    opacity: 0;
+}
+.imageAploader{
```
Comment (Suren-Sargsyan14) [inline src/Components/FileInput.js/index.module.scss]: same here

File: src/Components/FileInput.js/index.module.scss [hunk 29]
```diff
@@ -0,0 +1,39 @@
+.fileLoader {
+    display: flex;
+    flex-direction: column;
+    background-color: pink;
+}
+.fileLoaderComponent{
```
Comment (Suren-Sargsyan14) [inline src/Components/FileInput.js/index.module.scss]: same here

File: src/Components/Header/index.js [hunk 30]
```diff
@@ -1,21 +1,14 @@
...
-                name={"+"} 
-                func={openComponent}
-            />
-        </div>
-    );
-};
+const Header = ({openComponent}) => (
+    <div className={styles.header}>
+        <h1>Contact list</h1>
+        <Button  name={"Add new contact"} func={openComponent} />
```
Comment (Suren-Sargsyan14) [inline src/Components/Header/index.js]: please remove extra space

File: src/Components/NewContact/index.js [hunk 31]
```diff
@@ -1,11 +1,111 @@
...
+
+import { getDownloadURL, listAll, ref, uploadBytes } from "firebase/storage";
+import { v4 as uuidv4 } from 'uuid';
+
+import { fireStorage } from "../../Config/firebaseInit";
+import setContacts from "../../Services/setContacts";
+
+import styles from "./index.module.scss";
+
+const Form = ({openComponent, getContactFromBase}) => {
```
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.js]: same here

File: src/Components/NewContact/index.js [hunk 32]
```diff
@@ -1,11 +1,111 @@
...
+                            });
+                        });
+                    };
+                });
+            })
+        });
+    };
+
+    const handleSetContact = (e) => {
+        e.preventDefault();
```
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.js:57]: please remove extra space

File: src/Components/NewContact/index.js [hunk 33]
```diff
@@ -1,11 +1,111 @@
...
+      if(e.target[4].files[0]){
+          sendPicToStorage(e)
```
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.js]: please remove extra space

File: src/Components/NewContact/index.js [hunk 34]
```diff
@@ -1,11 +1,111 @@
...
+      }
+        setContacts(state);
+        getContactFromBase();
+        openComponent();
```
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.js:61]: please remove extra spaces

File: src/Components/NewContact/index.js [hunk 35]
```diff
@@ -1,11 +1,111 @@
...
-            New contact
-        </div>
+        <form onSubmit={handleSetContact} className={styles.newContact}>
+            <label className={styles.label}>
+                Name: 
+                <input
+                    type="text"
+                    name="firstName"
+                    value={state.firstName}
+                    onChange={handleInputChange}
```
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.js]: please remove extra spaces

File: src/Components/NewContact/index.js [hunk 36]
```diff
@@ -1,11 +1,111 @@
...
+                />
+            </label>
+            <label>
+                Surname: 
+                <input 
+                    type="text" 
+                    name="lastName"
+                    value={state.lastName}
+                    onChange={handleInputChange}
```
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.js]: please remove extra spaces

File: src/Components/NewContact/index.js [hunk 37]
```diff
@@ -1,11 +1,111 @@
...
+                />
+            </label>
+            <label>
+                Phone:
+                <input 
+                type="text"
+                name="phone"
+                value={state.phone} 
+                onChange={handleInputChange}
```
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.js]: please remove extra spaces

File: src/Components/NewContact/index.js [hunk 38]
```diff
@@ -1,11 +1,111 @@
...
+            />
+            </label>
+            <label>
+                E-mail:
+                <input 
+                    type="email" 
+                    name="email" 
+                    value={state.email}
+                    onChange={handleInputChange}
```
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.js]: please remove extra spaces

File: src/Components/NewContact/index.js [hunk 39]
```diff
@@ -1,11 +1,111 @@
...
+                />
+            </label>
+            <label>
+                Get photo 
+                <input type="file" name="avatar"/>
```
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.js]: please remove extra spaces

File: src/Components/NewContact/index.js [hunk 40]
```diff
@@ -1,11 +1,111 @@
...
+            </label>
+            <button type="submit">add contact</button>
+            <button onClick={(e)=>dontSetNewContact(e)}>go back</button>
```
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.js]: please set spaces before and after "=>" sign, and you could just write `<button onClick={dontSetNewContact}>`

File: src/Components/NewContact/index.js (hunk 35 above)
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.js]: Please render all you inputs with loop

File: src/Components/NewContact/index.module.scss [hunk 41]
```diff
@@ -0,0 +1,15 @@
+.newContact{
```
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.module.scss]: please add space before {

File: src/Components/NewContact/index.module.scss [hunk 42]
```diff
@@ -0,0 +1,15 @@
...
+    border: solid black 1px;
+    border-radius: 10px;
+    padding: 20px;
+    text-align: center;
+    height: 500px;
+    width: auto;
+    label{
```
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.module.scss]: same here

File: src/Components/NewContact/index.module.scss [hunk 43]
```diff
@@ -0,0 +1,15 @@
...
+        display: flex;
+        margin: 10px;
+        input{
```
Comment (Suren-Sargsyan14) [inline src/Components/NewContact/index.module.scss]: same here

File: src/Components/Pagination.js/index.js [hunk 44]
```diff
@@ -0,0 +1,27 @@
+import React, { memo } from "react";
+
+import styles from "./index.module.scss";
+
+const Pagination = ({contactPerPage, contactsQuantity, paginate}) => {
```
Comment (Suren-Sargsyan14) [inline src/Components/Pagination.js/index.js]: please set spaces before and after curly brackets

File: src/Components/Pagination.js/index.module.scss [hunk 45]
```diff
@@ -0,0 +1,13 @@
+.pagination{
```
Comment (Suren-Sargsyan14) [inline src/Components/Pagination.js/index.module.scss]: please add space before {

File: src/Components/Pagination.js/index.module.scss [hunk 46]
```diff
@@ -0,0 +1,13 @@
...
+    display: flex;
+    list-style-type: none;
+    >li{
```
Comment (Suren-Sargsyan14) [inline src/Components/Pagination.js/index.module.scss]: same here, and please add &

File: src/Components/Pagination.js/index.module.scss [hunk 47]
```diff
@@ -0,0 +1,13 @@
...
+        border: solid black 1px;
+        padding: 2px;
+        text-decoration: none;
+    }
+    >a{
```
Comment (Suren-Sargsyan14) [inline src/Components/Pagination.js/index.module.scss]: same here

File: src/Components/Search/index.js [hunk 48]
```diff
@@ -1,22 +1,29 @@
...
+        getFilterContacts(inputs);
     };
-
+    
     return (
         <div className={styles.search}>
-            <form onClick={(e)=>handleSearch}>
-                <input className={styles.input} type="text" placeholder="Get contact from your contact list..." />
-                <button type="submit" >Search</button>
+            <form onSubmit={(e)=>handleSearch(e)}>
```
Comment (Suren-Sargsyan14) [inline src/Components/Search/index.js]: you can just write `<form onSubmit={handleSearch}>`

File: src/Components/Search/index.js [hunk 49]
```diff
@@ -1,22 +1,29 @@
...
+                <input 
+                    name="filter"
+                    className={styles.input}
+                    type="text" 
+                    placeholder="Get contact from your contact list..."
+                    onChange={(e)=>setInputs(e.target.value)}
```
Comment (Suren-Sargsyan14) [inline src/Components/Search/index.js]: please add spaces before and after "=>" sign

File: src/Components/Search/index.module.scss [hunk 50]
```diff
@@ -1,6 +1,8 @@
 .search{
```
Comment (Suren-Sargsyan14) [inline src/Components/Search/index.module.scss]: please add space

File: src/Components/Search/index.module.scss [hunk 51]
```diff
@@ -1,6 +1,8 @@
...
     background-color: greenyellow;
+    align-items: center;
 }
 input{
```
Comment (Suren-Sargsyan14) [inline src/Components/Search/index.module.scss]: please add space

File: src/Components/index.js [hunk 52]
```diff
@@ -1,23 +1,88 @@
...
+import Pagination from "./Pagination.js";
+import { deleteObject, ref } from "firebase/storage";
 
 const Component = () => {
+    const { contacts, fillContacts } = useContext(ShowContacts);
     const [openNewContact, setStatusNewContact] = useState(false);
+    const [currentPage, setCurrentPage] = useState(1);
+    const [contactPerPage] = useState(5);
+
+    useEffect(() => getContactFromBase(),[]);
```
Comment (Suren-Sargsyan14) [inline src/Components/index.js]: please set space after semicolon, and use {}, also try to set all useEffects in the end of the methods

File: src/Components/index.js [hunk 53]
```diff
@@ -1,23 +1,88 @@
...
+        });
     };
 
-    const filterContacts = (tag) => {
-        console.log("filter")
+    const removeContact = async(id) => {
+        const desertRef = ref(fireStorage, `${id}/042684f3-218e-4975-89dc-24c55e826563`);
+        await deleteObject(desertRef)
+        .then(() => {
+          console.log("File deleted successfully")
```
Comment (Suren-Sargsyan14) [inline src/Components/index.js]: please remove consoles

File: src/Components/index.js [hunk 54]
```diff
@@ -1,23 +1,88 @@
...
-    const openComponent = () => {
-        setStatusNewContact(!openNewContact)
+// the function filtring only whole word not includes-must fix it
+    const getFilterContacts = tag => {
+        const fromFB = [];
+        const contactRef = collection (fireStore,"contacts");
+        const q = query (contactRef,
+            where ("firstName", "==", tag )
+        );
+        const querySnapshot = getDocs(q)
```
Comment (Suren-Sargsyan14) [inline src/Components/index.js]: please add newline

File: src/Components/index.js [hunk 55]
```diff
@@ -1,23 +1,88 @@
...
+    const openComponent = () => setStatusNewContact(!openNewContact);
 
-    const openComponent = () => {
-        setStatusNewContact(!openNewContact)
+// the function filtring only whole word not includes-must fix it
+    const getFilterContacts = tag => {
+        const fromFB = [];
+        const contactRef = collection (fireStore,"contacts");
+        const q = query (contactRef,
+            where ("firstName", "==", tag )
```
Comment (Suren-Sargsyan14) [inline src/Components/index.js]: don't set space after `where (` and before )

File: src/Components/index.js [hunk 56]
```diff
@@ -1,23 +1,88 @@
...
+
+    const openComponent = () => setStatusNewContact(!openNewContact);
 
-    const openComponent = () => {
-        setStatusNewContact(!openNewContact)
+// the function filtring only whole word not includes-must fix it
+    const getFilterContacts = tag => {
+        const fromFB = [];
+        const contactRef = collection (fireStore,"contacts");
+        const q = query (contactRef,
```
Comment (Suren-Sargsyan14) [inline src/Components/index.js]: don't set space after `query`

File: src/Components/index.js [hunk 57]
```diff
@@ -1,23 +1,88 @@
...
+    const paginate = page => setCurrentPage(page);
+
+    const openComponent = () => setStatusNewContact(!openNewContact);
 
-    const openComponent = () => {
-        setStatusNewContact(!openNewContact)
+// the function filtring only whole word not includes-must fix it
+    const getFilterContacts = tag => {
+        const fromFB = [];
+        const contactRef = collection (fireStore,"contacts");
```
Comment (Suren-Sargsyan14) [inline src/Components/index.js]: don't set space after `collection` and set space after semicolon

File: src/Services/setPhotosToUserData.js [hunk 58]
```diff
@@ -0,0 +1,10 @@
+import { fireStore} from "../Config/firebaseInit";
+
+import { doc, setDoc } from "firebase/firestore"; 
+
+const setPhotosToUserData = async(arrOfImages,id) => {
+    const cityRef = doc(fireStore, 'contacts', id);
+    await setDoc(cityRef, { imageArr: arrOfImages }, { merge: true });
+};
+
+export default setPhotosToUserData;
```
Comment (Suren-Sargsyan14) [inline src/Services/setPhotosToUserData.js]: please add a newline at the end of the file

File: src/Services/setPhotosToUserData.js [hunk 59]
```diff
@@ -0,0 +1,10 @@
+import { fireStore} from "../Config/firebaseInit";
+
+import { doc, setDoc } from "firebase/firestore"; 
+
+const setPhotosToUserData = async(arrOfImages,id) => {
```
Comment (Suren-Sargsyan14) [inline src/Services/setPhotosToUserData.js]: please add space after async and after semicolon

File: src/index.js [hunk 60]
```diff
@@ -1,13 +1,24 @@
...
 import './index.css';
+const Main = () => {
+    const [contacts, fillContacts] = useState([]);
+    
+    return (
+        <ShowContacts.Provider value={{contacts, fillContacts}}>
+           <App />
+        </ShowContacts.Provider>
+    );
+};
```
Comment (Suren-Sargsyan14) [inline src/index.js]: separate this with a file, and wrap main in app not vice versa

**Review by Suren-Sargsyan14 — CHANGES_REQUESTED** [review]

File: src/Components/FileInput.js/index.module.scss (hunk 26 above)
Comment (Shoxer90) [reply src/Components/FileInput.js/index.module.scss]: scss syntax would be like this 
.parentClass {
     color: green;
     childTag {
     any css...
     }
without & >

File: src/Components/Header/index.js (hunk 30 above)
Comment (Shoxer90) [reply src/Components/Header/index.js]: will I set spaces after and before { curlyes?

**Pull request merged**